app/helpers/flood_risk_engine/application_helper.rb

```diff
@@ -55,9 +55,5 @@ def page_title(title)
 
       title
     end
-
-    def submit_button_text(_enrollment)
```

Comment:
why remove this ? You may find you need it once the check details page comes along or definitly for the declaration page, as these pages require different wording of the burtton